.changeset/tame-chicken-mate.md

```diff
@@ -0,0 +1,5 @@
+---
+"@saleor/apps-otel": patch
+---
+
+Pass git commit sha as a version to Node SDK. This ensures that we can have version support for our monitoring service.
```

Comment:
suggestion: add info that this is opentelemetry NodeSDK

Reply:
This package is called "otel" ;)